spec/index.bs

```diff
@@ -1545,30 +1545,34 @@ interface GPUMapMode {
 
   **Returns:** {{ArrayBuffer}}
 
+  1. If |size| is unspecified, let |rangeSize| be this.{{[[size]]}} - |offset|. Otherwise, let |rangeSize| be |size|.
```

Comment:
We need a check that offset is less then this.size before we do that, otherwise it seems like it could skip the validation check for range being outside the buffer's bounds.